src/containers/Home.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Helmet } from "react-helmet"
+import './style.css'
+import twitter from '../../public/twitter.svg'
+import ReactSVG from '../../public/react.svg'
+import { createBackgroundPattern } from "./atoms.js"
+
+const Clicky = () => {
+  const clicky_site_ids = [];
+  clicky_site_ids.push(101142968);
+  const s = document.createElement('script');
+  s.type = 'text/javascript';
+  s.async = true;
+  s.src = '//static.getclicky.com/js';
+  ( document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0] ).appendChild( s );
+}
+
+
+export default class Home extends React.Component {
+
+  componentDidMount() {
+    // When the component finishes mounting, run the stuff from atom.js and Add clicky
+    createBackgroundPattern()
+    Clicky()
+    }
+
+  render(){
+    return (
+      <div>
+            <Helmet>
+              <title>This Week In React</title>
+              <meta charset="UTF-8" />
+              <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+              <meta name="twitter:card" content="summary_large_image" />
+              <meta name="twitter:site" content="@PhilippSpiess" />
+              <meta name="twitter:title" content="This Week In React ⚛️" />
+              <meta name="twitter:description" content="Weekly coverage of interesting pull requests, discussions, and RFCs around React Core and React DOM." />
+              <meta name="twitter:image" content="https://philippspiess.com/this-week-in-react/twitter-card.png" />
+              <link href="https://fonts.googleapis.com/css?family=Roboto:300,700" rel="stylesheet" />
+            </Helmet>
+            <div aria-hidden="true" className="atoms" />
+              <div className="main">
+                <section>
+                  <h1>This Week In React <img src={ ReactSVG } width= "60px"  alt="⚛️" id="logo" /></h1>
```

Comment:
Can you add an onClick event to the `<img/>` that also calls `createBackgroundPattern()`? This way we keep the current behavior of the easter egg 🙂

Reply:
👍